test/jaxp/javax/xml/jaxp/unittest/common/prettyprint/PrettyPrintTest.java

```diff
@@ -87,19 +92,32 @@
     // pretty-print=false, isStandalone=true, linebreak added after header
     private static final String XML_PPFALSE_LB
             = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<sometag/>";
+
+    private static final String XSL = "<?xml version=\"1.0\"?>\n"
+            + "<xsl:stylesheet version=\"1.0\" xmlns:xsl=\"http://www.w3.org/1999/XSL/Transform\">\n"
+            + "\n"
+            + "<!--Identity template, copies all content into the output -->\n"
+            + "    <xsl:template match=\"@*|node()\">\n"
+            + "        <xsl:copy>\n"
+            + "            <xsl:apply-templates select=\"@*|node()\"/>\n"
+            + "        </xsl:copy>\n"
+            + "    </xsl:template>\n"
+            + "\n"
+            + "</xsl:stylesheet>";
+
```

Comment:
Could be better to use text blocks?

Reply:
In general, we're keeping it at source level 8 to facilitate backports. As a result, only a few projects so far have been done using more advanced features.

Reply:
Makes sense. Thanks for the explanation.